assets/js/components/event/HlsPlayer.tsx

```diff
@@ -42,6 +43,12 @@ const HlsPlayer = ({ attachVideo, addMessage, presenterName, eventChannel, setCa
     };
   }, [addMessage]);
 
+  useEffect(() => {
+    if (!playerRef.current) return;
+    const interval = setInterval(() => doOCR(playerRef), 1000);
+    return () => clearInterval(interval);
+  }, [presenterName, attachVideo]);
```

Comment:
We could change whether the latency is measured in this way:

```
   const [measureHlsLatency, setMeasureHlsLatency] = useState<Boolean>(false);
  (window as any).toggleHlsLatency = () => setMeasureHlsLatency((prev) => !prev);
```

We add a function to a global scope, which is probably not very elegant, however it's simple and it works.


```suggestion
  useEffect(() => {
    if (!playerRef.current || !measureHlsLatency) return;
    const interval = setInterval(() => doOCR(playerRef), 1000);
    return () => clearInterval(interval);
  }, [presenterName, attachVideo, measureHlsLatency]);
```

This way we can start measuring latency by simply calling `toggleHlsLatency()` in the console.